components/script/dom/element.rs

```diff
@@ -2917,6 +2938,271 @@ impl ElementMethods for Element {
     fn AttachShadow(&self) -> Fallible<DomRoot<ShadowRoot>> {
         self.attach_shadow(IsUserAgentWidget::No)
     }
+
+    fn GetAriaAtomic(&self) -> Option<DOMString> {
+        self.get_nullable_string_attribute(&local_name!("aria-atomic"))
+    }
+    fn SetAriaAtomic(&self, value: Option<DOMString>) {
+        self.set_nullable_string_attribute(&local_name!("aria-atomic"), value);
+    }
+    fn GetAriaAutoComplete(&self) -> Option<DOMString> {
+        self.get_nullable_string_attribute(&local_name!("aria-autocomplete"))
```

Comment:
I think normally there are spaces between functions. Be sure to run `./mach fmt` before uploading your change -- though the lint didn't fail so maybe that isn't adding the spaces for some reason.